core/web/apiv2/observables.py

```diff
@@ -10,7 +10,7 @@
 
 for key in TYPE_MAPPING:
     if key in ["observable", "observables"]:
-        continue
+      continue
```

Comment:
```suggestion
        continue
```

Reply:
Fixed in 06d8d90bf3459c0d0295c76b00772dcc7c2d2ee2